.github/workflows/bundle_size.yml

```diff
@@ -54,6 +54,7 @@ jobs:
           pattern: "./frontend/.nuxt/dist/client/**/*.{modern.js,js,css}"
           exclude: "{./frontend/.nuxt/dist/client/lang-*.json.js,./frontend/.nuxt/dist/client/lang-*.json.modern.js}"
           minimum-change-threshold: 10
+          build-script: --filter frontend build
```

Comment:
This removes the need for the slightly confusing (in a now multi-buildable-package repo) root package.json `build` script.